guardian-ui/src/components/SetupComplete.tsx

```diff
@@ -0,0 +1,36 @@
+import { Flex, Heading, Text, Button, Icon } from '@chakra-ui/react';
+import React, { useCallback } from 'react';
+import { ReactComponent as ArrowRightIcon } from '../assets/svgs/arrow-right.svg';
+import { useGuardianContext } from '../hooks';
+import { SETUP_ACTION_TYPE } from '../types';
+
+export const SetupComplete: React.FC = () => {
+  const { dispatch } = useGuardianContext();
+
+  const handleContinue = useCallback(() => {
```

Comment:
If we've made it this far, should we SET_IS_SETUP_COMPLETE on mount? Why wait for the button click?